ui/src/components/ripple-out/RippleOutSpaceContainer.js

```diff
@@ -149,13 +149,13 @@ const RippleOutSpaceContainer = () => {
               placement="top"
             >
               <Box
-                top={['-6vh', '-14vh']}
-                left={['-30vw', '-18vw']}
+                top={['-6.5vh', '-17vh']}
```

Comment:
<img width="928" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/16095743/aa70a261-9d8c-417a-8178-ae2dcb51b867">
the dots now look misaligned to me? the building blocks dot is now out of the room and the office dot is covering the text

Reply:
![image](https://github.com/hmcc-global/hmcchk-web/assets/66002742/d710df13-6f1f-45b2-9583-5abe10d18834)
looks good on mine though lolol

Reply:
i guess i could make the buttons slightly smaller